scripts/run_stress_tests.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
```

Comment:
I think this script should be hoisted up to the root of the repo, alongside run_smoke.sh and smoke_loop.sh - until we decide to move them all somewhere.

Should run with cwd=the root of the repo without passing in bin and scripts args. (I did this, with some small mixups, but then ran into the size calculation problem noted below).

Reply:
Ok, moved it to the "root of the repo" and made necessary changes to paths.